support/test.go

```diff
@@ -69,12 +70,31 @@ func With(t *testing.T) Test {
 	}
 }
 
+func WithConfig(t *testing.T, cfg *rest.Config) Test {
```

Comment:
Can you adjust `With(t *testing.T)` to call this function, providing `nil` as cfg parameter? That will avoid code duplication.